Task3.c

```diff
@@ -0,0 +1,134 @@
+#include <stdio.h>
+#include <limits.h>
+#include <stdlib.h>
+
+
+void bubbleSort (int array[], int length)
```

Comment:
```suggestion
void bubbleSort(int array[], int length)
```
Откуда взялись пробелы между именем функции и её аргументами?